x-pack/legacy/plugins/apm/common/elasticsearch_fieldnames.ts

```diff
@@ -58,6 +58,10 @@ export const METRIC_JAVA_NON_HEAP_MEMORY_COMMITTED =
   'jvm.memory.non_heap.committed';
 export const METRIC_JAVA_NON_HEAP_MEMORY_USED = 'jvm.memory.non_heap.used';
 export const METRIC_JAVA_THREAD_COUNT = 'jvm.thread.count';
+export const METRIC_JAVA_GC_COUNT = 'jvm.gc.count';
+export const METRIC_JAVA_GC_TIME = 'jvm.gc.time';
+
+export const LABELS = 'labels';
```

Comment:
Shouldn't this be `LABELS_NAME = 'labels.name'`?

Reply:
Yeah, that's better. I used `${LABELS}.name` because I understood `labels` to be fully dynamic, and I was overthinking it.